lib/delivery.server.js

```diff
@@ -146,7 +152,7 @@ Delivery.prototype.subscribe = function() {
 
   //server sending to client
   this.socket.on('send.success', function(uid){
-    _this.pubSub.publish('send.success', uid);
+    _this.pubSub.publish('receive.success', uid);
```

Comment:
How come you're changing the event to a `receive` within the send event handler?